app/controllers/reservation_frames_controller.rb

```diff
@@ -2,7 +2,7 @@ class ReservationFramesController < ApplicationController
   before_action :signed_in_user, only: [:new, :create]
 
   def index
-    @reservation_frames = current_user&.reservation_frames
+    @reservation_frames = current_user && current_user.reservation_frames.after_current_time.sort_reserved_at_asc
```

Comment:
連続で&を使用でできるのはできるよ

```suggestion
    @reservation_frames = current_user&.reservation_frames&.after_current_time&.sort_reserved_at_asc
```

が、viewの方で定義した方が良かったのかもしれない。例えば`@reservation_frames`を使ってるビューで

```slim
- if current_user
  - current_user.reservation_frames.after_current_time.sort_reserved_at_asc.each do |reservation_frame|
```

とか

Reply:
わかりました viewに実装する方向でいきます